content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -7,7 +7,8 @@ capabilities.
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using the DVC Studio [REST API], `dvc get`, or DVC [Python API].
+versions using the DVC Studio [REST API], `dvc artifacts get`, or DVC [Python
+API].
```

Comment:
```suggestion
versions using the DVC Studio [REST API](/doc/studio/rest-api), `dvc artifacts get`, or DVC [Python
API](/doc/api-reference).
```

Unrelated, but these links are empty/broken now.

Reply:
Addressed in https://github.com/iterative/dvc.org/pull/5010/commits/126f337023bca5a3cbcf8efa56b9af69aa58efec!